### PR TITLE
Added styling for sponsor logos & first round of sponsors

### PR DESCRIPTION
**Pull request by sarj21**
JBHunt and Microstrategy added. Added 3 tiers of sizing in case we need them as sponsors come in

### CODE REVIEW

**Comment by sarj21** [comment]
FYI GitKraken rebase went a bit screwy so the commits doubled up and merged weird 😛

File: src/main.js [hunk 1]
```diff
@@ -3,11 +3,13 @@
 import Vue from 'vue';
 import smoothScroll from 'vue-smoothscroll';
 import VueAnalytics from 'vue-analytics';
+import VueLazyload from 'vue-lazyload';
 import App from './App.vue';
 import router from './router';
 
 Vue.config.productionTip = false;
 Vue.use(smoothScroll);
+Vue.use(VueLazyload);
```
Comment (mecolmg) [inline src/main.js]: Is that really it??? That's remarkably easy! 😄

File: src/components/SponsorsSection.vue [hunk 2]
```diff
@@ -30,12 +47,57 @@ export default {
 
 #sponsors-section {
   width: 100%;
-  min-height: 300px;
+  height: auto;
```
Comment (mecolmg) [inline src/components/SponsorsSection.vue]: You could just remove this line, as auto is the default value.

File: src/components/SponsorsSection.vue [hunk 3]
```diff
@@ -30,12 +47,57 @@ export default {
...
+
+.sponsor-tier{
+  align-self: center;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-bottom: 25px;
+
+  img {
+    width: auto;
```
Comment (mecolmg) [inline src/components/SponsorsSection.vue]: All these width/height: auto should not be necessary.

**Comment by mecolmg** [comment]
So yeah, check out the commit I just sent, if you like it lgtm, otherwise just revert it 👍

**Comment by sarj21** [comment]
Your commit looks good. Let me test it with multiple images and see how it looks

**Pull request merged**